packages/kbn-i18n/src/react/index.tsx

```diff
@@ -18,9 +18,7 @@
  */
 
 import { InjectedIntl as _InjectedIntl, InjectedIntlProps as _InjectedIntlProps } from 'react-intl';
-
-export type InjectedIntl = _InjectedIntl;
-export type InjectedIntlProps = _InjectedIntlProps;
+export type { InjectedIntl, InjectedIntlProps } from 'react-intl';
```

Comment:
Wow, I can't rationalize how this is actually different from the previous code... This seems like a bug in TS and I'm glad you hear you plan to report it to the TS team... I'm shocked that you figured this out.

Reply:
Very odd indeed.